src/style.css

```diff
@@ -1,3 +1,103 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+
+  --blue-link: #00acdc;
+}
+
+.body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  display: inline-block;
+  height: 40px;
+  width: 40px;
+}
+
+.header__nav {
+  display: block;
+}
+
+.header__nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+}
+
+.header__nav__item-1 {
+  padding-right: 20px;
```

Comment:
With this use of classes your code is more prone to errors and difficult to change, so group them more

Reply:
fixed